### DISCUSSION TITLE
Commodity Support

### DISCUSSION THREAD

**Discussion by frederik042853** [Fifteen15Studios/MMM-PreciousMetals#1] [General]
Love to see support for commodities especially Aluminum

**Comment by Fifteen15Studios** [comment]
Added

**Comment by frederik042853** [comment]
Thanks

**Comment by frederik042853** [comment]
Sorry for the additional edit.
Performed git pull (results already have the latest version)
Copied exactly your example 2 just added "aluminum"
Result shows all except aluminum?

Any help is much appreciated

**Reply by Fifteen15Studios** [reply]
It should be working. Can you copy your exact config for this module so I can take a look at it?

**Reply by Fifteen15Studios** [reply]
Also, I just remembered, you may need a reboot before it works. Not the whole raspberry pi, but completely closing out of MM and reloading it, instead of just reloading the window. This reboot will load the new node-helper, whereas reloading the page will not.
